ceresdb/client_test.go

```diff
@@ -28,12 +28,12 @@ func now() int64 {
 func build2Rows() ([]*types.Row, error) {
 	rows := make([]*types.Row, 0, 2)
 
-	curr := now()
+	timestamp := int64(1668124800000)
```

Comment:
ceresdb will discard rows older than 7 days by default, so this couldn't be a fixed value